content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md

```diff
@@ -0,0 +1,92 @@
+### INTRODUCTION
+Technology is fast growing in the world today. This recency in growth is enhancing the livelihoods of humans and their environment. This includes the building of many self-driven cars and some smart digital systems.
+A few problems in the understanding of its working algorithm have arisen. 
+   
+   These problems have triggered developers to ask questions like: "
+
+* What is the design algorithm enhancing deep learning?
+* Why are recent developed systems able to recognize humans and their environment?
+* How can devices move accurately without human beings controlling them?
+
+      An excessive volume of information (data sets) can be trained. This uses learning algorithms in deep learning. Deep learning systems tend to gather functional facts from larger datasets. This experience (learning) is in relation to human ways of getting smarter. 
+
```

Comment:
Add `Table of contents` section (Make sure to use markdown hyperlinks) and `Pre-requisities` section.

Have a look at other articles